openapi.yaml

```diff
@@ -0,0 +1,592 @@
+  /v1/fhir/metadata:
+    get:
+      operationId: fhirMetaData
+      description: ''
+      security: []
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+            application/fhir+json:
+              schema:
+                type: array
+                items: {}
+          description: ''
+      tags:
+        - v1
+  /v1/connect/userinfo:
+    get:
+      operationId: listopenidconnect_userinfos
+      description: ''
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+          description: ''
+      tags:
+        - v1
+  '/v1/fhir/Patient/{resource_id}':
+    get:
+      operationId: listReadViewPatients
+      description: ''
+      parameters:
+        - name: resource_id
+          in: path
+          required: true
+          description: ''
+          schema:
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+            application/fhir+json:
+              schema:
+                type: array
+                items: {}
+          description: ''
+      tags:
+        - v1
+  /v1/fhir/Patient/:
+    get:
+      operationId: listSearchViewPatients
+      description: ''
+      parameters:
+        - in: query
+          name: identifier
+          schema:
+            type: string
+          required: false
+          description: 'The patient identifier to search for'
+        - in: query
+          name: _lastUpdated
+          schema:
+            type: string
+          required: false
+          description: 'Include resources last updated in the given range'
+        - in: query
+          name: startIndex
+          schema:
+            type: string
+          required: false
+          description: 'The offset used for result pagination'
+        - in: query
+          name: _has:Coverage
+          schema:
+            type: string
+          required: false
+          description: 'Part D coverage type/year'
+        - in: query
+          name: cursor
+          schema:
+            type: string
+          required: false
+          description: 'The cursor used for result pagination'
+        - in: query
+          name: _id
+          schema:
+            type: string
+          required: false
+          description: 'The patient identifier to search for'
+        - in: query
+          name: _lastUpdated
+          schema:
+            type: string
+          required: false
+          description: 'Include resources last updated in the given range'
+        - in: query
+          name: startIndex
+          schema:
+            type: string
+          required: false
+          description: 'The offset used for result pagination'
```

Comment:
@nbragdon Was seeing some duplicate errors showing in the Swagger editor. These two params look to be defined above. 

Here was the full list form the messages.  All seem to be for the v[1|2]/fhir/patient resources.

```
Errors
Hide
 
Structural error at paths./v1/fhir/Patient/.get.parameters
should NOT have duplicate items (items ## 2 and 7 are identical)
i: 7
j: 2
Jump to line 90
Semantic error at paths./v1/fhir/Patient/.get.parameters.6
Sibling parameters must have unique name + in values
Jump to line 126
Semantic error at paths./v1/fhir/Patient/.get.parameters.7
Sibling parameters must have unique name + in values
Jump to line 132
Structural error at paths./v2/fhir/Patient/.get.parameters
should NOT have duplicate items (items ## 2 and 7 are identical)
i: 7
j: 2
Jump to line 377
Semantic error at paths./v2/fhir/Patient/.get.parameters.6
Sibling parameters must have unique name + in values
Jump to line 413
Semantic error at paths./v2/fhir/Patient/.get.parameters.7
Sibling parameters must have unique name + in values
Jump to line 419
```

Reply:
Oops, duplicates removed 👍